### PR TITLE
link to storybook from README.md to close #192

### LINKED ISSUE #192: documentation: add link to storybook somewhere on this repository's main page
I think it would be convenient if somewhere on [the main page of this repository](https://github.com/amilajack/freesound-player) we had a link to this repository's storybook at https://amilajack.github.io/freesound-player.

### PR DESCRIPTION
**Pull request by hamirmahal**
This pull request closes #192 if we merge it by linking to this repository's storybook from this repository's `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,19 @@ cd tide
...
-
 # E2E Tests
 yarn start
 # Run this in a separate terminal
 yarn cypress open
 ```
+
+## Storybook
+
+https://tide-app.github.io/tide/
```
Comment (amilajack) [inline README.md:38]: Wrong url to storybook?

**Review by amilajack — CHANGES_REQUESTED** [review]

**Comment by amilajack** [comment]
I think its better to publicly host the storybook and point to the publicly hosted version

File: README.md (hunk 1 above)
Comment (hamirmahal) [reply README.md:38]: Are you sure?

https://user-images.githubusercontent.com/43425812/141656270-f0dbb9a3-f558-48b8-a880-9dc6a63b2789.mov

**Comment by hamirmahal** [comment]
> I think its better to publicly host the storybook and point to the publicly hosted version

I'm not sure what you mean by this. We're publicly hosting the `Storybook` at https://tide-app.github.io/tide/, no?

**Comment by amilajack** [comment]
Ah nvm I see it now

**Review by amilajack — APPROVED** [review]

**Pull request merged**